show_options.mk

```
@@ -0,0 +1,115 @@
+show_build_options:
+	echo " PLATFORM = $(PLATFORM)"
```

Inside recipes (lines starting with a tab character), shell syntax is used, so `$()` creates a subshell. That's not what you want here; you want parameter substitution (aka. interpolation). Use `$var` or `${var}` instead.
```suggestion
	echo " PLATFORM = $PLATFORM"
```
Make this change to all the lines below as well.